components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```diff
@@ -729,6 +730,7 @@ def generic_start_worker(
         "-e", f"CLP_CONFIG_PATH={container_clp_config.logs_directory / container_config_filename}",
         "-e", f"CLP_LOGS_DIR={container_logs_dir}",
         "-e", f"CLP_LOGGING_LEVEL={worker_config.logging_level}",
+        "-e", f"WORKER_LOG_PATH={container_worker_log_path}",
```

Comment:
* Prefix with CLP_ to avoid conflicts with other env vars?
* I think we discussed in a previous PR that we don't really need to have `WORKER` in the name, but feel free to disagree in this case.

Reply:
I feel this is a special case where worker has two logs, one for clp executable and one for worker itself. I would keep worker in this case so it's clear